core/src/main/contraband-scala/sbt/librarymanagement/DeliverConfiguration.scala

```diff
@@ -0,0 +1,58 @@
+/**
+ * This code is generated using [[http://www.scala-sbt.org/contraband/ sbt-contraband]].
+ */
+
+// DO NOT EDIT MANUALLY
+package sbt.librarymanagement
+final class DeliverConfiguration private (
+  val deliverIvyPattern: Option[String],
+  val status: Option[String],
+  val configurations: Option[scala.Vector[String]],
```

Comment:
Could we introduce a `ConfigRef` (or similar) type, so these aren't just bare, indistinguishable strings?